OpenOversight/app/main/views.py

```diff
@@ -608,9 +608,9 @@ def edit_department(department_id):
 def list_officer(
     department_id,
     page=1,
-    race=[],
-    gender=[],
-    rank=[],
+    race=None,
+    gender=None,
+    rank=None,
```

Comment:
Why this change?

Reply:
This falls into the ["mutable default arguments" problem](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments).